sample/manifest.yml

```diff
@@ -5,10 +5,15 @@ applications:
 #
 - name: logging-sample-app
   instances: 1
+  buildpack: sap_java_buildpack
+  memory: 256M
+  disk_quota: 256M
   path: target/logging-sample-app-3.7.0.war
   env:
     RANDOM_SLEEP: true
 # Set LOG_*: true to activate logging of respective field
     LOG_SENSITIVE_CONNECTION_DATA: false
     LOG_REMOTE_USER: false
     LOG_REFERER: false
+    JAVA_OPTS: '-XX:ReservedCodeCacheSize=40M -XX:MaxMetaspaceSize=28M -Xss256K'
+    # Default: '-XX:ReservedCodeCacheSize=240M -XX:MaxMetaspaceSize=141152K -Xss1M'
```

Comment:
comment above code please

Reply:
Is there really no restrictions for the heap space?